cfme/common/physical_server_views.py

```diff
@@ -0,0 +1,188 @@
+    """Represents physical toolbar and its controls."""
+    monitoring = Dropdown(text="Monitoring")
+    configuration = Dropdown(text="Configuration")
+    policy = Dropdown(text="Policy")
+    power = Dropdown(text="Power")
+    identify = Dropdown(text="Identify")
+    lifecycle = Dropdown(text="Lifecycle")
+
+    @ParametrizedView.nested
+    class custom_button(ParametrizedView):  # noqa
+        PARAMETERS = ("button_group", )
+        _dropdown = Dropdown(text=Parameter("button_group"))
+
+        def item_select(self, button, handle_alert=False):
+            self._dropdown.item_select(button, handle_alert=handle_alert)
+
+
+class PhysicalServerDetailsEntities(View):
+    """Represents Details page."""
+    properties = SummaryTable(title="Properties")
+    networks = SummaryTable(title="Networks")
+    relationships = SummaryTable(title="Relationships")
+    power_management = SummaryTable(title="Power Management")
+    assets = SummaryTable(title="Assets")
+    firmware = SummaryTable(title="Firmware")
+    network_devices = SummaryTable(title="Network Devices")
+    smart = SummaryTable(title="Smart Management")
+
+
+class PhysicalServerDetailsView(ComputePhysicalInfrastructureServersView):
+    """Main PhysicalServer details page."""
+    breadcrumb = BreadCrumb(locator='.//ol[@class="breadcrumb"]')
+    toolbar = View.nested(PhysicalServerDetailsToolbar)
+    entities = View.nested(PhysicalServerDetailsEntities)
+
+    @property
+    def is_displayed(self):
+        title = "{name} (Summary)".format(name=self.context["object"].name)
+        return (self.in_compute_physical_infrastructure_servers and
+                self.breadcrumb.active_location == title)
+
+
+class PhysicalServerTimelinesView(TimelinesView, ComputePhysicalInfrastructureServersView):
+    """Represents a PhysicalServer Timelines page."""
+
+    @property
+    def is_displayed(self):
+        return (self.in_compute_physical_infrastructure_servers and
+                super(TimelinesView, self).is_displayed)
+
+
+class PhysicalServerManagePoliciesView(BaseLoggedInPage):
+    """PhysicalServer's Manage Policies view."""
+    policies = BootstrapTreeview("protectbox")
+    entities = View.nested(BaseNonInteractiveEntitiesView)
+    save_button = Button("Save")
+    reset_button = Button("Reset")
+    cancel_button = Button("Cancel")
+
+    @property
+    def is_displayed(self):
+        return False
+
+
+class PhysicalServersToolbar(View):
+    """Represents hosts toolbar and its controls."""
+    configuration = Dropdown(text="Configuration")
+    policy = Dropdown(text="Policy")
+    lifecycle = Dropdown(text="Lifecycle")
+    monitoring = Dropdown(text="Monitoring")
+    power = Dropdown(text="Power")
+    identify = Dropdown(text="Identify")
+    view_selector = View.nested(ItemsToolBarViewSelector)
+
+
+class PhysicalServerSideBar(View):
+    """Represents left side bar. It usually contains navigation, filters, etc."""
+    pass
```

Comment:
**Required**

Since this is empty, I think I'd prefer it left out of the views for now, with a #TODO to add a sidebar with an accordion for the filter - or just add the accordion widget now from widgetastic.patternfly.

Reply:
@mshriver Just making sure that I understand. Do you mean that the _PhysicalServerSideBar_ class should be derived from _Accordion_ instead of _View_?  Maybe similar to the _filters_ class in _cfme/common/host_views.py_ ?

Reply:
Ah sorry for confusion - no, the sidebar should still inherit from View - but it should nest accordion views inside of it. In this case just the one filter accordion, on most pages there are a few accordions within the sidebar. 

Example:
In cfme.cloud.keypairs, there is a `KeyPairDetailsAccordion` class inheriting `View`, and that class is `View.nested()` in `KeyPairDetailsView` as the `sidebar` widget:
https://github.com/ManageIQ/integration_tests/blob/master/cfme/cloud/keypairs.py#L34
https://github.com/ManageIQ/integration_tests/blob/master/cfme/cloud/keypairs.py#L97

You can directly nest the view with a decorator, instead of creating a separate class, if you'd like. Since its only a single accordion it would be short.